### PR TITLE
Azure Workload Cluster Release 16.0.1

### PR DESCRIPTION
**Pull request by T-Kukawka**

### CODE REVIEW

File: azure/v16.0.1/README.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
+# :zap: Giant Swarm Release v16.0.1 for Azure :zap:
+
+This Workload CLuster release upgrades kubernetes to 1.21.5 in order to fix [CVE-2021-25741](https://github.com/kubernetes/kubernetes/issues/104980) where Symlink Exchange Can Allow Host Filesystem Access.
```
Comment (whites11) [inline azure/v16.0.1/README.md:3]:
```suggestion
This Workload Cluster release upgrades Kubernetes to 1.21.5 in order to fix [CVE-2021-25741](https://github.com/kubernetes/kubernetes/issues/104980) where Symlink Exchange Can Allow Host Filesystem Access.
```

File: azure/v16.0.1/announcement.md [hunk 2]
```diff
@@ -0,0 +1 @@
+**Workload cluster release v16.0.1 for Azure** is now available. This release upgrades kubernetes version to 1.21.5. Further details can be found in the [release notes](https://docs.giantswarm.io/changes/workload-cluster-releases-azure/releases/azure-v16.0.1/).
```
Comment (whites11) [inline azure/v16.0.1/announcement.md:1]:
```suggestion
**Workload cluster release v16.0.1 for Azure** is now available. This release upgrades Kubernetes version to 1.21.5. Further details can be found in the [release notes](https://docs.giantswarm.io/changes/workload-cluster-releases-azure/releases/azure-v16.0.1/).
```

**Review by fiunchinho — APPROVED** [review]

**Review by whites11 — APPROVED** [review]

**Comment by whites11** [comment]
> Azure e2e green

remember to test upgrade as well using `/test upgrade-azure`

**Comment by whites11** [comment]
> > Azure e2e green
> 
> remember to test upgrade as well using `/test upgrade-azure`

Lol I triggered it and it badly failed. looking

**Pull request merged**